en-US/Design.xml

```diff
@@ -1058,6 +1058,11 @@ STEP 14: COMMIT <emphasis>...output omitted...</emphasis> localhost/nexus:latest
 
 <!-- 		<listitem>
 			 </listitem>		-->
+				<para>
+					Do not use possessives to refer to product names.
```

Comment:
I spoke with Julian about this, and we're going to wait on guidance around possessives and company names. Because of that, I rolled back this suggestion. Sorry about that!